src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/CurrentUserIdentityProvider.Unix.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Http
+{
+    internal static class CurrentUserIdentityProvider
+    {
+        public static string GetIdentity()
+        {
+            return string.Empty;
```

Comment:
We don't need to do anything per user on unix?

Reply:
I had a long discussion with @alnikola about this scenario on Linux. It is not clear how to get the identity because it is not a first-class thing on Linux compared with Windows. It is very specific to Kerberos tickets cached (either in temp file cache or memory). There are also no known scenarios where impersonation really works on Linux since threads can't just change Kerberos identities with a Linux API call. There is an orthogonal aspect to the Kerberos tickets which aren't associated with threads.

So, I suggested that since we don't any real scenario we can construct right now for this, that we just basically use `string.Empty`.

Reply:
Ok, thanks.